tests/appsTest.js

```diff
@@ -1,229 +1,168 @@
-describe('Main: Currently testing apps management,', function () {
-  const utf8 = require('utf8')
-  const config = require('./config/config.json')
-
-  // LIBRARY INSTANCE
-  let oc
+import { MatchersV3, XmlBuilder } from '@pact-foundation/pact/v3'
 
-  // TESTING CONFIGS
+describe('Main: Currently testing apps management,', function () {
   const nonExistentApp = 'nonExistentApp123'
 
   // PACT setup
-  const Pact = require('@pact-foundation/pact-web')
-  const provider = new Pact.PactWeb()
   const {
     validAuthHeaders,
-    CORSPreflightRequest,
     capabilitiesGETRequestValidAuth,
     GETRequestToCloudUserEndpoint,
-    ocsMeta,
     xmlResponseHeaders,
-    pactCleanup,
+    createProvider,
     createOwncloud
   } = require('./pactHelper.js')
 
-  const responseBody = function (data) {
-    return '<?xml version="1.0"?>\n' +
-    '<ocs>\n' +
-    ocsMeta('ok', '100') +
-    data +
-    '</ocs>'
+  const changeAppStatus = (provider, action) => {
+    const method = action === 'enable' ? 'POST' : 'DELETE'
+    return provider
+      .uponReceiving(action + ' apps')
+      .withRequest({
+        method,
+        path: MatchersV3.regex(
+          /.*\/ocs\/v1\.php\/cloud\/apps\/.+$/,
+          '/ocs/v1.php/cloud/apps/files'
+        ),
+        headers: validAuthHeaders
+      })
+      .willRespondWith({
+        status: 200,
+        headers: xmlResponseHeaders,
+        body: new XmlBuilder('1.0', '', 'ocs').build(ocs => {
+          ocs.appendElement('meta', '', (meta) => {
+            meta
+              .appendElement('status', '', 'ok')
+              .appendElement('statuscode', '', 100)
+              .appendElement('message', '', '')
+          }).appendElement('data', '', '')
+        })
+      })
   }
 
-  beforeAll(function () {
-    const promises = []
-    promises.push(provider.addInteraction(CORSPreflightRequest()))
-    promises.push(provider.addInteraction(capabilitiesGETRequestValidAuth()))
-    promises.push(provider.addInteraction(GETRequestToCloudUserEndpoint()))
-    // a request to GET attributes of an app
-    const attributes = {
```

Comment:
please make a note in the PR about why all these interactions are not needed

Reply:
Tests using these interactions were removed in https://github.com/owncloud/owncloud-sdk/pull/615